zephyr/wrapper.c

```diff
@@ -186,13 +194,17 @@ const char irq_name_level5[] = "level5";
 
 /*
  * CAVS IRQs are multilevel whereas BYT and BDW are DSP level only.
+ *
+ * For i.MX we use the IRQ_STEER
  */
 int interrupt_get_irq(unsigned int irq, const char *cascade)
 {
 #if CONFIG_SOC_SERIES_INTEL_ADSP_BAYTRAIL ||\
 	CONFIG_SOC_SERIES_INTEL_ADSP_BROADWELL || \
-	CONFIG_LIBRARY
+	CONFIG_LIBRARY || \
+	CONFIG_IMX
```

Comment:
I would leave CONFIG_LIBRARY as the last CONFIG in this line as it special. Not a real platform.

Reply:
Done